src/khoj/routers/helpers.py

```diff
@@ -550,21 +558,38 @@ async def text_to_image(
                 )
                 image = response.data[0].b64_json
 
+            with timer("Convert image to webp", logger):
+                # Convert png to webp for faster loading
+                decoded_image = base64.b64decode(image)
+                image_io = io.BytesIO(decoded_image)
+                png_image = Image.open(image_io)
+                webp_image_io = io.BytesIO()
+                png_image.save(webp_image_io, "WEBP")
+                webp_image_bytes = webp_image_io.getvalue()
+                webp_image_io.close()
+                image_io.close()
+
             with timer("Upload image to S3", logger):
-                image_url = upload_image(image, user.uuid)
-            return image, status_code, improved_image_prompt, image_url
+                image_url = upload_image(webp_image_bytes, user.uuid)
+            if image_url:
+                intent_type = "text-to-image-v2"
+            else:
+                intent_type = "text-to-image-v3"
+                image = base64.b64encode(webp_image_bytes).decode("utf-8")
```

Comment:
Let's use an enum for these image types and include descriptions of what they imply? Hard to follow and maintain the code otherwise.